File: Easy/226. Invert Binary Tree.cpp
/**
 * Definition for a binary tree node.
 * struct TreeNode {
 *     int val;
 *     TreeNode *left;
 *     TreeNode *right;
 *     TreeNode(int x) : val(x), left(NULL), right(NULL) {}
 * };
 */
class Solution {
public:
    TreeNode* invertTree(TreeNode* root) {
        swapChild(root);
		return root;
    }

	void swapChild(TreeNode* node){
		if(node == NULL) return;
		TreeNode* tmp = node->left;
		node->left = node->right;
		node->right = tmp;
		swapChild(node->left);
		swapChild(node->right);
	}
};

File: Easy/226. Invert Binary Tree.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, x):
#         self.val = x
#         self.left = None
#         self.right = None

class Solution:
    def invertTree(self, root: TreeNode) -> TreeNode:
        self.swapChild(root)
        return root
        
    def swapChild(self, node: TreeNode) -> None:
        if not node: return
        node.left, node.right = node.right, node.left
        self.swapChild(node.left)
        self.swapChild(node.right)